projects/config-validator/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/GoogleCloudPlatform/config-validator/"
+main_repo: "https://github.com/GoogleCloudPlatform/config-validator/"
+primary_contact: "martin.p.petkov@gmail.com"
+auto_ccs:
+- morgante.pell@morgante.net
```

Comment:
@MartinPetkov Can we use my work email instead? `morgantep@google.com`

Reply:
Of course, done.